File: CenturyFromYear/CenturyFromYear.py
import math

def centuryFromYear(year):
    if year % 100 == 0:
        div = year / 100
        print(div)
        roundDown = math.floor(div)
        return roundDown
    else:
        div = year / 100 + 1
        return math.floor(div)

File: CenturyFromYear/centuryfromyear..c
#include <stdio.h>
#include <math.h>

int centuryFromYear(int year){
    if (year % 100 == 0){
        int div = year / 100;
        return div;
    }
    else{
        int div = year/100 + 1;
        return floor(div);
    }
}